node-launchpad/src/components/status.rs

```diff
@@ -591,6 +632,40 @@ impl Component for Status<'_> {
                     }
                 }
             },
+            Action::OptionsActions(OptionsActions::UpdateNodes) => {
+                debug!("Got action to Update Nodes");
+                self.load_node_registry_and_update_states()?;
+                if self.lock_registry.is_some() {
+                    error!(
+                        "Registry is locked ({:?}) Cannot Update nodes now. Stop them first.",
+                        self.lock_registry
+                    );
+                    return Ok(None);
+                } else {
+                    debug!("Lock registry ({:?})", self.lock_registry);
+                };
+                debug!("Setting lock_registry to UpdatingNodes");
+                self.lock_registry = Some(LockRegistryState::UpdatingNodes);
+                let action_sender = self.get_actions_sender()?;
+                info!("Got action to update nodes");
+                let _ = self.update_node_items(Some(NodeStatus::Updating));
+                let (service_names, peer_ids) = self.get_service_names_and_peer_ids();
+
+                let upgrade_nodes_args = UpgradeNodesArgs {
+                    action_sender,
+                    connection_timeout_s: 5,
+                    do_not_start: true,
+                    custom_bin_path: None,
+                    force: false,
+                    fixed_interval: None,
+                    peer_ids,
+                    provided_env_variables: None,
+                    service_names,
```

Comment:
We can use a empty `vec![]` here and we'd upgrade all the nodes. We don't have to pass in the service names.

Reply:
No harm with the current impl, so resolving this.

Reply:
Follow up PR :)